app/factories/flood_risk_engine/steps/form_object_factory.rb

```diff
@@ -42,6 +42,7 @@ def setup_form_object(step)
           begin
             form_name.constantize
           rescue NameError => x
```

Comment:
That's interesting! It must be a dummy app thing. This is what you get from the same page when you mount the engine via the dummy app:
![missing_translation_error](https://cloud.githubusercontent.com/assets/213040/15360442/769b35b8-1d05-11e6-8765-617e5674ce10.png)